pkg/scaffold/helm/watches.go

```diff
@@ -36,12 +37,15 @@ func (s *WatchesYAML) GetInput() (input.Input, error) {
 	}
 	s.HelmChartsDir = HelmChartsDir
 	s.TemplateBody = watchesYAMLTmpl
+	if s.ChartName == "" {
+		s.ChartName = s.Resource.LowerKind
+	}
 	return s.Input, nil
 }
 
 const watchesYAMLTmpl = `---
 - version: {{.Resource.Version}}
   group: {{.Resource.FullGroup}}
   kind: {{.Resource.Kind}}
-  chart: /opt/helm/{{.HelmChartsDir}}/{{.Resource.LowerKind}}
+  chart: /opt/helm/{{.HelmChartsDir}}/{{.ChartName}}
```

Comment:
According to the [chart name conventions](https://github.com/helm/helm/blob/master/docs/chart_best_practices/conventions.md#chart-names), chart names SHOULD be lower case. However, the directory name MUST match the name in Chart.yaml. It seems the chart name case is not enforced, but the directory matching  the name in Chart.yaml is.

```console
$ helm create Test
Creating Test

$ helm package Test
Successfully packaged chart and saved it to: /tmp/Test-0.1.0.tgz

$ mv Test test

$ helm package test
Error: directory name (test) and Chart.yaml name (Test) must match
```

Since our code works by using Helm's packages to load the chart metadata, `.ChartName` is effectively a read-only field provided by Helm. For that reason, I think I lean toward accepting what Helm gives us without modification. I would expect Helm to error out if a user tried to create or fetch an invalid chart.